app/frontend/src/cljs/teet/ui/panels.cljs

```diff
@@ -198,10 +198,10 @@
 (defn button-with-modal [{:keys [modal-title button-component modal-component]}]
   (r/with-let [open? (r/atom false)
                open! #(reset! open? true)
-               close! #(reset! open? false)]
+               close! #(do (reset! open? false) nil)]
```

Comment:
The do seems unnecessary

Reply:
It is necessary, I want to explicitly return nil